AdobeStockImage/etc/Api/AssetRepositoryInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Api;
+
+use Magento\AdobeStockImage\Api\Data\AssetInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+interface AssetRepositoryInterface
+{
+    /**
+     * Save asset
+     * @api
+     * @param AssetInterface $item
+     * @return bool
+     */
+    public function save(AssetInterface $item) : bool;
+
+    /**
+     * Get new empty object
+     * @return AssetInterface
+     */
+    public function getNewEmptyModel() : AssetInterface;
```

Comment:
I don't think this method is necessary, a factory can be used to create a blank model.

Reply:
Method deleted